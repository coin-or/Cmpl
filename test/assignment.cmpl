%data : machines set, locations set, combinations set[2], c[combinations] 

variables:
	x[combinations]: binary;

objectives:
	costs: sum{  k in combinations : c[k]*x[k] } -> min ;

constraints:
	sos_m { i in machines: sum{ j in locations, [i,j] << combinations  : x[i,j] } = 1; }
	sos_l { j in locations: sum{ i in machines , [i,j] << combinations  : x[i,j] } <= 1; }
