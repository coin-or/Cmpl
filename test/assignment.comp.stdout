Objective value:  29.0
Objective status:  optimal
x[1,4]   1   0.0   1.0
x[2,3]   1   0.0   1.0
x[3,1]   1   0.0   1.0
sos_m[1]   1.0   1.0   1.0
sos_m[2]   1.0   1.0   1.0
sos_m[3]   1.0   1.0   1.0
sos_l[1]   1.0   -inf   1.0
sos_l[3]   1.0   -inf   1.0
sos_l[4]   1.0   -inf   1.0
